### DISCUSSION TITLE
2005/06/30/dll%E7%9A%84%E5%AF%BC%E5%85%A5%E5%BA%93%E4%B8%AD%E6%98%AF%E4%BB%80%E4%B9%88%E4%BF%A1%E6%81%AF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#38] [Announcements]
# 2005/06/30/dll%E7%9A%84%E5%AF%BC%E5%85%A5%E5%BA%93%E4%B8%AD%E6%98%AF%E4%BB%80%E4%B9%88%E4%BF%A1%E6%81%AF

当DLL被链接时，链接程序要查找关于输出变量，函数，或C++类的信息，并自动生成一个lib文件。该lib文件包含一个DLL输出的符号列表。如果要链接引用该DLL的输出符号的任何可执行模块，该lib文件是必不可少的（使用GetProcAddress除外）。

https://blog.qizhen.xyz/2005/06/30/dll%E7%9A%84%E5%AF%BC%E5%85%A5%E5%BA%93%E4%B8%AD%E6%98%AF%E4%BB%80%E4%B9%88%E4%BF%A1%E6%81%AF

**Comment by ruanqizhen** [comment]
♡Angie♡
2005年07月1日 19:43 编辑
wow,好专业的知识哦,飘飘学习ing…

回复
yumi
2005年06月30日 15:32 编辑
呵呵，路过

回复
